src/components/module_parser.js

```diff
@@ -119,7 +125,19 @@ const ModuleParser = {
             return null;
         }
     },
-
+    /**
+     *  check if a module is valid
+     *  the modulename should look like this: I.BA_ANLS.F1901
+     */
+    validateModule: (hsluModule) => {
+        const moduleNameRegex = /^(\w+)\.(.+)\.[FH]\d{4}/g;
```

Comment:
This is hard to read and according to that regex `a..........F0000` is valid, which is questionable.